File: 10/CODING/HTML/07/rect.js
const readline = require('node:readline');

const rl = readline.createInterface({
    input : process.stdin,
    output : process.stdout,
});

let width, height;

rl.question(`Input width : `, widthInput => {
    width = parseInt(widthInput); // Number
    rl.question(`Input height: `, heightInput => {
        height = parseInt(heightInput);

        const area = width * height;
        console.log(`The area is ${area}`);

        rl.close();
    })
   
})

File: 10/INFORMATIKA/C++/10-11/05/rect.cpp
#include <iostream>
using namespace std;

double find_area(double length, double width);

int main()
{
    cout << "AREA = " << find_area(10, 20) << endl;
}

double find_area(double length, double width)
{
    double area = length * width;
    return area
}